loader/debug_utils.c

```diff
@@ -89,7 +89,7 @@ VkBool32 util_SubmitDebugUtilsMessageEXT(const struct loader_instance *inst, VkD
 
     if (NULL != pCallbackData) {
         VkLayerDbgFunctionNode *pTrav = inst->DbgFunctionHead;
-        VkDebugReportObjectTypeEXT object_type = VK_OBJECT_TYPE_UNKNOWN;
+        VkDebugReportObjectTypeEXT object_type = (VkDebugReportObjectTypeEXT)VK_OBJECT_TYPE_UNKNOWN;
```

Comment:
This should just really be: `VK_DEBUG_REPORT_OBJECT_TYPE_UNKNOWN_EXT`.  I'm surprised GCC 8.2 didn't catch this.

Reply:
Yep, pushed update